src/components/Form/Form.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { FormTypes } from '../../types/FormTypes';
+
+import style from './Form.module.scss';
+import imgDollar from '../../images/icon-dollar.svg';
+import imgPerson from '../../images/icon-person.svg';
+
+export const Form: React.FC<FormTypes> = ({
+  handleBillAmtInput,
+  handleSelectedTip,
+  billAmt,
+  handleSelectedTipButton,
+  setNumbersOfPeople,
+  numOfPeople,
+  errorOfNumbers,
+}) => {
+  return (
+    <div className={style.form}>
+      <div className={style.label__group}>
+        <label className={style.label}>Bill</label>
+        <input
+          type="number"
+          id="bill"
+          onInput={handleBillAmtInput}
+          value={billAmt}
+          className={style.input__error}
+        />
+        <img src={imgDollar} alt="" className={style.icons} />
+      </div>
+      <div className={style.select__tip}>
+        <label className={style.label}>Selected Tip %</label>
+        <div className={style.tip__amount__wrapper}>
+          <button className={style.tip__btn} onClick={() => handleSelectedTipButton(5)}>
```

Comment:
also, you can move this to a separate component and create an array of available tip percentages and just use map